main.cpp

```diff
@@ -18,16 +18,23 @@ using namespace clang::ast_matchers;
 using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
+private:
+    Rewriter& rewriter_;
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
+    CastCallBack(Rewriter& rewriter) : rewriter_(rewriter) {}
+
+    void run(const MatchFinder::MatchResult& Result) override {
+        const auto* styleCastExpr = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");

```

Comment:
What if `styleCastExpr` is a null pointer? How to protect the program from crashing in such cases?

Reply:
To protect the program from crashing in such cases, it is necessary to add this check for the code below: 
if (styleCastExpr != nullptr) {

}